ingestion-beam/src/test/java/com/mozilla/telemetry/schemas/JSONSchemaStoreTest.java

```diff
@@ -39,6 +39,8 @@ public void testDocTypeExists() {
     JSONSchemaStore store = JSONSchemaStore.of(SCHEMAS_LOCATION, EMPTY_ALIASING_CONFIG_LOCATION);
     assertTrue(store.docTypeExists("telemetry", "main"));
     assertTrue(store.docTypeExists("telemetry", "update"));
+    assertTrue(store.docTypeExists("telemetry", "untrustedModules"));
```

Comment:
Shouldn't we test BigQuerySchemaStore? The new case we're resolving is with e.g. `bookmarks_sync` on BQ output. For test purposes they amount to the same thing I suppose (in the current implementation, but I have thoughts on that in the MPS PR).

Reply:
The BigQueryIntegrationTest does effectively test this behavior for the BigQuerySchemaStore, but I can understand pushback for more unit testing.